mii/server.py

```diff
@@ -74,6 +74,15 @@ def _create_lb_config(self, mii_config):
         replica_pool = _allocate_processes(mii_config.hostfile,
                                            mii_config.tensor_parallel,
                                            mii_config.replica_num)
+        if mii_config.deploy_rank!=-1 and mii_config.replica_num==1:
+            if isinstance(mii_config.deploy_rank, int):
+                replica_pool = [(replica_pool[0][0], [mii_config["deploy_rank"]])]
```

Comment:
@tulika612 - could you fix merge conflicts and we can work on getting this PR reviewed if you still think it is valuable?